derp/derphttp/derphttp_test.go

```
@@ -385,7 +397,7 @@ func TestBreakWatcherConn(t *testing.T) {
 		}
 		remove := func(m derp.PeerGoneMessage) { t.Logf("remove: %v", m.Peer.ShortString()); peers-- }
 
-		watcher1.RunWatchConnectionLoop(ctx, serverPrivateKey1.Public(), t.Logf, add, remove)
+		watcher1.RunWatchConnectionLoop(ctx, serverPrivateKey1.Public(), t.Logf, add, remove, noopNotifyError)
```

:question: Do you want to ignore errors or do you want to fail on error?